index.js

```diff
@@ -104,8 +104,18 @@ async function getHdsComponentsList() {
 }
 
 console.log('fetching repos data which contain hds- in package.json');
-const fetchData = await fetcher(`${githubApiUrl}/search/code?q=hds-+in:file+filename:package.json+org:${owner}&per_page=1000&page=1`);
-const searchData = await fetchData.json();
+
+// initialize with total_count 1 to get into the loop
+let searchData = { items: [], total_count: 1 };
+let page = 1;
+
+while (searchData.items.length < searchData.total_count) {
+    console.log('fetching page', page);
```

Comment:
There is no error handling, but I guess the while loop breaks when async/await fails?

I'd add catch() to both.

Reply:
Ok I'll do it

Reply:
Yeah the loop should break but yeah, better to add it too...